tasks/main.yml

```diff
@@ -50,7 +51,6 @@
 - name: Set JBoss EAP Service Files Facts
   set_fact:
     jboss_eap_service_conf_file: "{{ jboss_eap_service_conf_dir }}/{{ jboss_eap_user }}.conf"
-    jboss_eap_runtime_conf_file: "{{ jboss_eap_home }}/bin/standalone.conf"
```

Comment:
the Issue #12 does not touch this line at all, I did try clean builds multiple times on my Vagrant Vm's it went through